src/Tribe/Integrations/Freemius.php

```diff
@@ -96,6 +96,51 @@ public function __construct() {
 		add_action( 'admin_init', [ $this, 'maybe_remove_activation_complete_notice' ] );
 	}
 
+	/**
+	 * When should we load Freemius to users
+	 *
+	 * @since  TBD
+	 *
+	 * @param  integer $threshold Percentage of which we will load Freemius
+	 *
+	 * @return boolean
+	 */
+	public function should_load( $threshold = 10 ) {
+		$previous_versions = Tribe__Settings_Manager::get_option( 'previous_ecp_versions', [] );
```

Comment:
is this checking for ECP or TEC?

Reply:
That was the string that was set way back when it was created. But it's right